src/current/cockroachcloud/troubleshooting-page.md

```diff
@@ -40,7 +40,7 @@ Failed running "sql"
 
 <h4>Solution: CA certificate conflicts</h4>
 
-If you have existing certificates in `~/.cockroach-certs` used to connect to CockroachDB {{ site.data.products.core }} or CockroachDB {{ site.data.products.advanced }} clusters and are trying to connect to a CockroachDB {{ site.data.products.basic }} or {{ site.data.products.standard }} cluster using [`cockroach sql`](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/cockroach-sql), you need download the CA cert by running the command from the **Cluster Overview** > **Connect** dialog if you have not already done so, and then set the `sslrootcert` parameter in the connection string you use when running `cockroach sql`.
+If you have existing certificates in `~/.cockroach-certs` used to connect to CockroachDB {{ site.data.products.core }} or CockroachDB {{ site.data.products.advanced }} clusters and are trying to connect to a CockroachDB {{ site.data.products.standard }} or {{ site.data.products.basic }} cluster using [`cockroach sql`]({% link {{site.current_cloud_version}}/cockroach-sql.md %}), you need to download the CA cert by running the command from the **Cluster Overview** > **Connect** dialog if you have not already done so, and then set the `sslrootcert` parameter in the connection string you use when running `cockroach sql`.
```

Comment:
the reordering of standard and basic is not in #18875

Reply:
Correct, I flipped them on purpose because we are supposed to highlight Standard before Basic.